### DISCUSSION TITLE
Kanal

### DISCUSSION THREAD

**Discussion by cryptopus58** [IBAX-io/go-ibax#876] [Ideas]
Daha fazla bilgi için ulaşabileceğimiz telegram youtube vs tarzı kanallar mevcut mu

**Comment by Skips1** [comment]
Millet anlatıyor kanka Araştırabilirsin Ama benim gördüğüm kadarıyla İ bax ın sosyal medya Adminleri biraz pasif :(

**Comment by hsynkader** [comment]
https://t.me/IBAXTurkish resmi ibax Türkiye kanalı.
